tests/acceptance/features/apiContract/sharesReport.feature

```diff
@@ -82,3 +82,66 @@ Feature: REPORT request to Shares space
       | dav-path-version |
       | old              |
       | new              |
+
+
+  Scenario Outline: check the REPORT response of the found security folder
+    Given using <dav-path-version> DAV path
+    And user "Alice" has created folder "/secureFolder"
+    And user "Alice" has uploaded file with content "secure content" to "/secureFolder/secure.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | secureFolder  |
+      | space           | Personal      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Secure viewer |
+    When user "Brian" searches for "secureFolder" using the WebDAV API
+    Then the HTTP status code should be "207"
+    # And the following headers should match these regular expressions
+    #   | X-Request-Id | /^[a-zA-Z]+\/[a-zA-Z]+\.feature:\d+(-\d+)?$/ |
+    And the "REPORT" response to user "Brian" should contain a mountpoint "secureFolder" with these key and value pairs:
```

Comment:
do we need to check the request-id header?

Reply:
I forgot to enable it